### PR TITLE
(chore) Separated the address fields into a new object in Formik context

### PR DESCRIPTION
**Pull request by vasharma05**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [x] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

This PR separates the address fields into a new object in the Formik context.
The reason behind this is that the address fields were not sent with the patient object properly.
I'm not sure if this the reason behind it, but this task is something to be taken care of.

Link to the thread: https://openmrs.slack.com/archives/CHP5QAE5R/p1665143528651379?thread_ts=1664967610.888199&cid=CHP5QAE5R

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-appointments-app/translations/en.json [hunk 1]
```diff
@@ -117,6 +117,7 @@
   "selectVisitType": "Please select a Visit Type",
   "service": "Service",
   "serviceType": "Service Type",
+  "serviceWorkloadTitle": "",
```
Comment (hadijahkyampeire) [inline packages/esm-appointments-app/translations/en.json:120]: An empty translation 🤔

**Review by hadijahkyampeire — APPROVED** [review]

File: packages/esm-patient-registration-app/src/patient-registration/input/dummy-data/dummy-data-input.component.tsx [hunk 2]
```diff
@@ -23,16 +23,18 @@ export const dummyFormValues: FormValues = {
...
-  address2: '',
-  cityVillage: 'Recife',
-  stateProvince: 'Pernambuco',
-  country: 'Brazil',
-  postalCode: '50030-310',
   isDead: false,
   deathDate: '',
   deathCause: '',
   relationships: [],
+  address: {
```
Comment (rbuisson) [inline packages/esm-patient-registration-app/src/patient-registration/input/dummy-data/dummy-data-input.component.tsx:30]: @vasharma05 , does this assume the address fields are static, hardcoded?

Comment (vasharma05) [reply packages/esm-patient-registration-app/src/patient-registration/input/dummy-data/dummy-data-input.component.tsx:30]: No @rbuisson, these are just the default values to these fields. Other fields can be added dynamically, if they are present in the form, handled by Formik.

File: packages/esm-appointments-app/translations/en.json (hunk 1 above)
Comment (vasharma05) [reply packages/esm-appointments-app/translations/en.json:120]: It came out after extracting translations.

**Comment by vasharma05** [comment]
Thank you @hadijahkyampeire !!!

**Pull request merged**